ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -23,3 +28,77 @@ def __init__(self, resource_name="", *args, **kwargs):
         super(StorageCluster, self).__init__(
             resource_name=resource_name, kind='StorageCluster', *args, **kwargs
         )
+
+
+def add_capacity(capacity_string):
+    """
+        Add storage capacity to the cluster
+
+        Args:
+            capacity_string(int): Size of the storage to add
+        Returns:
+            True if capacity was added, False if not
+
+    """
+
+    ocp = OCP(namespace=defaults.ROOK_CLUSTER_NAMESPACE, kind=constants.STORAGECLUSTER)
+    osd_capacity = parse_size_to_int(get_storage_cluster()['spec']['storageDeviceSets']
+                                        [0]['dataPVCTemplate']['spec']
+                                        ['resources']['resources']['storage'])
+
+    osd_replicas = parse_size_to_int(get_storage_cluster()['spec']['storageDeviceSets'][0]['replicas'])
+    worker_nodes = len(helpers.get_worker_nodes())
+    current_osd_count = len(pod.get_osd_pods())
+    available_osd_number = worker_nodes * 3 - current_osd_count
```

Comment:
replace worker_nodes * 3 - current_osd_count with "(worker_nodes * 3) - current_osd_count" to improve readability.

Reply:
Done